sfr/abtr_xsgen_workflow/xs_generation_mesh.i

```diff
@@ -0,0 +1,38 @@
+# ==============================================================================
+# ABTR Equivalent-full core cross section generation mesh
+# Application: Griffin
+# POC: Shikhar Kumar (kumars at anl.gov)
+# If using or referring to this model, please cite as explained in
+# https://mooseframework.inl.gov/virtual_test_bed/citing.html
+# ==============================================================================
+
```

Comment:
and this?
```suggestion
!include !abtr_het_mesh.i
```
or not?

seems easier than the multiple input files

Reply:
Cool, this works. Let me update to this

Reply:
I'd also need to update the documentation to describe the updated run command and that this command just imports the referenced .i file right?

Reply:
Documentation is updated to describe this process